src/main/java/org/jenkinsci/test/acceptance/po/WizardCustomizeJenkins.java

```diff
@@ -94,8 +94,26 @@ public void startInstall() {
         clickButton("Install");
     }
 
-    public void deselectAll() {
+    public void selectAll() {
+        clickLink("All");
+        elasticSleep(200);
```

Comment:
the use of elasticSleep is a sure sign that this test is flaky.
What are you waiting for to complete please wait for that to happen rather than just some random amount of time.